eng/Versions.props

```diff
@@ -14,13 +14,14 @@
     <SdkFeatureBand>$(VersionMajor).$(VersionMinor).$(VersionSDKMinor)00</SdkFeatureBand>
     <BuiltinWorkloadFeatureBand>$(VersionMajor).$(VersionMinor).100</BuiltinWorkloadFeatureBand>
     <VersionPrefix>$(VersionMajor).$(VersionMinor).$(VersionSDKMinor)$(VersionFeature)</VersionPrefix>
+    <LatestVersionPrefix>10.0.100</LatestVersionPrefix>
```

Comment:
The name is confusing since latest would be 11 and the value is 10. Maybe make it <PreviewVersionPrefix>?